Sources/Build/ManifestBuilder.swift

```diff
@@ -478,7 +478,7 @@ extension LLBuildManifestBuilder {
         _ target: SwiftTargetBuildDescription,
         inputs: [Node],
         cmdOutputs: [Node]
-    ) {
+    ) throws {
```

Comment:
Is this `throws` still needed, now that the code in the function no longer throws?